web/marketing/pages/referral-offers.html

```diff
@@ -11,50 +11,39 @@
         <div class="col col-heading">Redeemable&nbsp;Capacity</div>
         <div class="col col-heading">Created</div>
         <div class="col col-heading">Expiration</div>
-        <div class="col col-heading"></div>
-    </div>
-    <hr>
+        <div class="col col-heading">Status</div>
+    </div><hr>
     <div class="row offer-heading ">
-        <p class="offer-type">DEFAULT&nbsp;OFFER</p>
-    </div>
-    <div class="row data-row">
-        <div class="col ml-3">May Referral</div>
-        <div class="col">$50</div> 
-        <div class="col">$50</div>
-        <div class="col">48</div>
-        <div class="col">200</div>
-        <div class="col">05/01/19</div>
-        <div class="col">06/01/19</div>
-        <div class="col"><strong>Live</strong></div>
-        <div class="col col-heading edit-offer">EDIT</div>
+        <p class="offer-type">default&nbsp;offer</p>
     </div>
-    <hr>
+    {{range .ReferralOffers}}
+        {{if eq .Status 2}}
+            <div class="row data-row">
+                <div class="col ml-3">{{.Name}}</div>
+                <div class="col">${{.InviteeCreditInCents}}</div> 
+                <div class="col">${{.AwardCreditInCents}}</div>
```

Comment:
Why is the credit in cents? However $ indicates it's in non-cents? Or is it something else than dollars?